pkgs/games/domination/default.nix

```diff
@@ -83,6 +84,11 @@ in stdenv.mkDerivation {
     runHook postInstall
   '';
 
+  preFixup = ''
+    # remove extra metadata files for jar files
```

Comment:
Perhaps explain why they're removed: for determinism.

Reply:
Sorry, for "not breaking stripJavaArchivesHook".

Reply:
I'll put it into the comment, just in case